client/verta/verta/registry/entities/_model.py

```diff
@@ -101,6 +114,14 @@ def get_or_create_version(self, name=None, desc=None, labels=None, attrs=None, t
         id : str, optional
             ID of the Model Version. This parameter cannot be provided alongside `name`, and other
             parameters will be ignored.
+        input_description : str, optional
+            Input Description of the Model Version.
+        hide_input_label : bool, optional
+            Hide Input Label of the Model Version.
```

Comment:
It feels strange to me for `hide_input_label` and `hide_output_label` to be in the client/CLI, because showing labels is purely a UI feature, right? 🤔 

If we must have them in the client, I would recommend

```suggestion
        hide_input_label : bool, default False
            Hide Input Label of the Model Version.
```

instead of `optional` (`=None`) for booleans.